components/root_portal.jsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+export default class RootPortal extends React.Component {
```

Comment:
This comes directly from this example here: https://reactjs.org/docs/portals.html#event-bubbling-through-portals